Overview.src.html

```diff
@@ -3391,21 +3456,23 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   <dl class=switch>
    <dt><code data-anolis-spec=fileapi>Blob</code>
    <dd>
-    <p>Push a copy of <var>object</var>'s contents to <var>stream</var>.
+    <p><span title=concept-enqueue-ReadableStream>Enqueue</span> a <code>Uint8Array</code> object
```

Comment:
pipeTo what though? We got rid of the idea of a writable stream for requests.

I think the way to do this is to be vague. It also does need to be done in parallel. Hmm. Something like:

In parallel, enqueue a sequence of `Uint8Array` objects representing _object_'s contents, as they become available while reading the blob.